lib/modules/datasource/terraform-provider/index.spec.ts

```diff
@@ -68,7 +75,24 @@ describe('modules/datasource/terraform-provider/index', () => {
       httpMock
         .scope(primaryUrl)
         .get('/v1/providers/hashicorp/azurerm')
-        .reply(200, JSON.parse(consulData))
+        .reply(200, JSON.parse(azurermData))
+        .get('/.well-known/terraform.json')
+        .reply(200, serviceDiscoveryResult);
+      const res = await getPkgReleases({
+        datasource: TerraformProviderDatasource.id,
+        depName: 'azurerm',
+      });
+      expect(res).toMatchSnapshot();
+      expect(res).not.toBeNull();
```

Comment:
```suggestion
      expect(res).toMatchSnapshot();
```

use `toMatchObject` and something like `{ registryUrl: '.., ', ..., releases: expect.toBeArrayOfObjects().toBeArrayOfSize(30) }`

Reply:
is there a reason that snapshot testing has fallen out of favour? this is tests that I've not changed.